### ISSUE TITLE
Got an error with one or more lambda function with node.js 14 runtime

### ISSUE THREAD

**Issue by manjulaK** [awslabs/aws-dataall#721]
### Describe the bug

Received an email with the following : "We are contacting you as we have identified that your AWS Account currently has one or more Lambda functions using the Node.js 14 runtime." would like to know if you are going to fix the lambdas impacted? This seems to be for all data.all accounts.

### How to Reproduce

```
*P.S. Please do not attach files as it's considered a security risk. Add code snippets directly in the message body as much as possible.*
```

### OS

All

### Python version

3.1

### AWS data.all version

all

**Comment by wolfit** [comment]
Duplicate of #708 and #709 and fixed in PR #710.
Please review #710 
I suggest to close this.

**Comment by dlpzx** [comment]
Also addressed in #655
